be/src/olap/rowset/beta_rowset.cpp

```diff
@@ -271,6 +271,63 @@ Status BetaRowset::link_files_to(const std::string& dir, RowsetId new_rowset_id,
     return Status::OK();
 }
 
+Status BetaRowset::link_files_without_index_to(const std::string& dir, RowsetId new_rowset_id,
```

Comment:
It's almost the same as link_files_to. It can be merged to link_files_to with default argument.